client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionDetailsFields/InteractionDetailsFields.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {useMemo} from 'react';
+import {Controller, useFormContext} from 'react-hook-form';
+import FormInput from 'commons/FormInput/FormInput';
+import AlphanumericTextField from 'commons/AlphanumericTextField/AlphanumericTextField';
+import InteractionEventDialogFields
+    from 'models/enums/InteractionsEventDialogContext/InteractionEventDialogFields';
+import {getOptionsByPlaceAndSubplaceType} from 'Utils/ContactEvent/placeTypesCodesHierarchy';
+import TimeForm from './InteractionTimeForm';
```

Comment:
Fix order of imports list
separate commons/models imports from files imports and modules imports

Reply:
Hopefully fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/commit/bd9413139437352056d0aaff99583d726152f8a1)